sdk/examples/wallet/nft_collection/01_mint_collection_nft.rs

```diff
@@ -79,3 +91,37 @@ async fn main() -> Result<()> {
 
     Ok(())
 }
+
+fn get_immutable_metadata(index: usize, issuer_nft_id: NftId) -> String {
+    format!(
+        r#"{{
+        "standard":"IRC27",
+        "version":"v1.0",
+        "type":"video/mp4",
+        "uri":"ipfs://wrongcVm9fx47YXNTkhpMEYSxCD3Bqh7PJYr7eo5Ywrong",
+        "name":"Shimmer OG NFT #{index}",
+        "description":"The Shimmer OG NFT was handed out 1337 times by the IOTA Foundation to celebrate the official launch of the Shimmer Network.",
+        "issuerName":"IOTA Foundation",
+        "collectionId":"{issuer_nft_id}",
+        "collectionName":"Shimmer OG"
+    }}"#
+    )
```

Comment:
Doesn't this include the newlines?
If so they should be replaced at the end, they just require more bytes and therefore a higher storage deposit which is not wanted

Reply:
ah, I see, well ... I remove the newlines then.

Reply:
Done

Reply:
Hm will also add the empty spaces in between
```Rust
fn main() {
    let r = format!(
        r#"{{
        "standard":"IRC27",
        "version":"v1.0",
        "type":"video/mp4"
    }}"#
    )
    .replace('\n', "");

    println!("{r}");

    println!(
        "{}",
        serde_json::from_str::<serde_json::Value>(&r)
            .unwrap()
            .to_string()
    );
}
```
will print
```
{        "standard":"IRC27",        "version":"v1.0",        "type":"video/mp4"    }
{"standard":"IRC27","type":"video/mp4","version":"v1.0"}
```
so I suggest to use `serde_json::from_str::<serde_json::Value>()`

Reply:
well, okay, but there's only empty space in your example, not in the actual example. Your suggestion still makes sense though.